app/models/draft_vaccination_record.rb

```diff
@@ -296,6 +293,31 @@ def reset_unused_fields
     end
   end
 
+  def earliest_possible_value
+    session.academic_year.to_academic_year_date_range.first.beginning_of_day
```

Comment:
I think it's possible for `session` to be `nil` if a historical record is being edited, in which case we can skip the validation.